scm/doc/TechGuide/chap_cases.rst

```diff
@@ -505,10 +505,10 @@ Optional arguments:
 Examples to run from within the ``scm/etc/scripts`` directory to create SCM cases starting
 with the output from a UFS Weather Model regression test(s):
 
-On the supported platforms Cheyenne (NCAR) and Hera (NOAA), there are
+On the supported platforms Derecho (NCAR) and Hera (NOAA), there are
 staged UWM RTs located at:
 
--  Cheyenne ``/glade/scratch/epicufsrt/GMTB/CCPP-SCM/UFS_RTs``
+-  Derecho ``/glade/scratch/epicufsrt/GMTB/CCPP-SCM/UFS_RTs``
```

Comment:
It looks like this directory was never ported over from Cheyenne before it was scrubbed. @dustinswales Does it make sense to grab some examples from the WM baselines Derecho and ask EPIC to stage them in a similar Derecho directory? Does it matter which tests we use?

Reply:
Changed to more generic `/glade/derecho/scratch/epicufsrt/FV3_RT/` which does exist